behave/helper/helper_base.py

```diff
@@ -16,10 +16,11 @@ def __init__(self, driver):
         self._driver = driver
         self.call_centre_backend = Backend("/call_centre/api/v1/")
         self.call_centre_backend.authenticate(**CALL_CENTRE_ZONE)
+        self.date_now = None
```

Comment:
Change this to `self.date_start_this_month = datetime.datetime.now().date().replace(day=1, month=date_now.month - 1)`

Reply:
Changed.